ui/components/multichain/account-list-item/account-list-item.js

```diff
@@ -128,7 +129,7 @@ const AccountListItem = ({
   // scroll the item into view
   const itemRef = useRef(null);
   useEffect(() => {
-    if (selected) {
+    if (selected && shouldScrollToWhenSelected) {
       itemRef.current?.scrollIntoView?.();
     }
   }, [itemRef, selected]);
```

Comment:
Minor: can you add `shouldScrollToWhenSelected` to the dependencies to rm the eslint warning?

Reply:
Done, thanks for catching that!